### ISSUE TITLE
Error: Unexpected token `target` in `parse_target`

### ISSUE THREAD

**Issue by abreis** [chancancode/rust-delegate#2]
The following minimal example:

``` rust
#[macro_use]
extern crate delegate;
extern crate petgraph;
use petgraph::{Graph, Directed, graph::DefaultIx};

pub struct DirectedAcyclicGraph<N, E>(Graph<N, E, Directed, DefaultIx>);

impl<N, E> DirectedAcyclicGraph<N, E> {
    delegate! {
        target self.0 {
            /// Return the total number of nodes in the graph.
            pub fn node_count(&self) -> usize;
        }
    }
}
```

Fails to compile with the error:

```
error: no rules expected the token `target`
  --> src/main.rs:10:9
   |
10 |         target self.0 {
   |         ^^^^^^

error: ParseError! Unexpected token `target` in `parse_target`. This is possibly a bug in the delegate library itself. Please file an issue with the following debug information:

{ items : [  ] , action : delegate__expand }


  --> src/main.rs:9:5
   |
9  | /     delegate! {
10 | |         target self.0 {
11 | |             /// Return the total number of nodes in the graph.
12 | |             pub fn node_count(&self) -> usize;
13 | |         }
14 | |     }
   | |_____^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

This is `rustc 1.26.0` on `stable-x86_64-apple-darwin`.

**Comment by chancancode** [comment]
Good catch! Fixed in 0.1.2, thanks for the bug report!

**Comment by abreis** [comment]
Thanks for the quick fix!
